base/ca/src/org/dogtagpki/ct/sct/SCTProcessor.java

```diff
@@ -0,0 +1,103 @@
+//
+// Copyright Red Hat, Inc.
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+//
+
+package org.dogtagpki.ct.sct;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.dogtagpki.ct.LogServer;
+import org.dogtagpki.server.ca.CAEngine;
+import org.dogtagpki.server.ca.CAEngineConfig;
+
+import com.netscape.certsrv.base.EBaseException;
+import com.netscape.certsrv.base.EPropertyNotFound;
+import com.netscape.certsrv.base.IConfigStore;
+
+/**
+ * @author Dinesh Prasanth M K
+ *
+ */
+public class SCTProcessor {
+    public static org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(SCTProcessor.class);
+
+    protected IConfigStore mConfig;
+
+    public void init() throws Exception {
+        CAEngine engine = CAEngine.getInstance();
+        CAEngineConfig cs = engine.getConfig();
+
+        mConfig = cs.getCAConfig().getSubStore("certTransparency");
+    }
+
+    public boolean isCTEnabled() throws EPropertyNotFound, EBaseException {
+        return mConfig.getBoolean("enable");
+    }
+
+    /**
+     * Read log server configuration from CA's CS.cfg
+     *
+     * Example:
+     *
+     * <pre>
+     * {@code
+     * ca.certTransparency.enable=true
+     * ca.certTransparency.log.1.enable=true
+     * ca.certTransparency.log.1.pubKey=MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEw8i8S7qiGEs9NXv0ZJFh6uuOmR2Q7dPprzk9XNNGkUXjzqx2SDvRfiwKYwBljfWujozHESVPQyydGaHhkaSz/g==
+     * ca.certTransparency.log.1.url=http://ct.googleapis.com:80/testtube/
+     * ca.certTransparency.log.1.version=1
+     * ca.certTransparency.log.2.enable=false
+     * ca.certTransparency.log.2.pubKey=MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEKATl2B3SAbxyzGOfNRB+AytNTGvdF/FFY6HzWb+/HPE4lJ37vx2nEm99KYUy9SoNzF5VyTwCQG5nL/c5Q77yQQ==
+     * ca.certTransparency.log.2.url=http://ct.googleapis.com:80/logs/crucible/
+     * ca.certTransparency.log.2.version=1
+     * ca.certTransparency.log.num=3
```

Comment:
Isn't the num supposed to be 1 as only one is enabled?  Question: if you put two, would it still be fine as the 2nd one is not enabled?

Reply:
Answering here for future reference.

In this particular case,  it can be either 1 or 2. But, in general, `ca.certTransparency.log.num` represents "The total number of CT log entries defined, both enabled and disabled"